src/services/followers/followers.ts

```
@@ -0,0 +1,110 @@
+        followingId,
+      },
+    });
+
+    logger.info(`User ${followerId} successfully followed user ${followingId}`);
+
+    return {
+      message: 'Followed successfully',
+      followerId,
+      followingId,
+    };
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)